config/default.schema.json

```diff
@@ -1559,12 +1573,6 @@
 		}
 	},
 	"required": [],
-	"allOf": [
-		{
-			"$ref": "#/definitions/FEATURE_ES_MERLIN_ENABLED",
-			"$ref": "#/definitions/ANTIVIRUS"
-		}
-	],
 	"definitions": {
 		"FEATURE_USER_MIGRATION_ENABLED": {
```

Comment:
can be removed as well since FEATURE_USER_MIGRATION_SYSTEM_ID is per default an empty string, thus this never fails, also meaning that you can remove FEATURE_USER_MIGRATION_ENABLED from your configmap "overwrites"